src/shell/shell.cpp

```diff
@@ -1222,13 +1222,13 @@ void SHELL_Init() {
 			"Displays or changes file attributes.\n");
 	MSG_Add("SHELL_CMD_ATTRIB_HELP_LONG",
 	        "Usage:\n"
-	        "  \033[32;1mattrib\033[0m \033[37;1m[ATTRIBUTES]\033[0m \033[36;1mPATTERN\033[0m [/S]\n"
+	        "  [color=green]attrib[reset] [color=white][ATTRIBUTES][reset] [color=cyan]PATTERN[reset] [/S]\n"
```

Comment:
Does using bold `[b]...[/b]` instead of `[color=white]...[reset]` work? Looking at `dev_con.h`, it would appear to be supported.

Reply:
[b]…[/b] will probably work if the initial color is normal white. But this may not be always the case though. So the meaning of [color=white] and [b] may be different.